cfme/tests/cli/test_appliance_cli.py

```diff
@@ -91,37 +91,37 @@ def test_appliance_console_cli_external_create(
     unconfigured_appliance_secondary.wait_for_web_ui()
 
 
-@pytest.mark.uncollect('No IPA servers currently available')
 @pytest.mark.parametrize('auth_type', ['sso_enabled', 'saml_enabled', 'local_login_disabled'],
     ids=['sso', 'saml', 'local_login'])
-def test_external_auth(auth_type, ipa_crud, app_creds):
+def test_appliance_console_cli_external_auth(auth_type, ipa_crud, app_creds, configured_appliance):
     evm_tail = LogValidator('/var/www/miq/vmdb/log/evm.log',
                             matched_patterns=['.*{} to true.*'.format(auth_type)],
-                            hostname=ipa_crud.hostname,
+                            hostname=configured_appliance.hostname,
                             username=app_creds['sshlogin'],
-                            password=app_creds['password'])
+                            password=app_creds['sshpass'])
     evm_tail.fix_before_start()
-    command = 'appliance_console_cli --extauth-opts="/authentication/{}=true"'.format(auth_type)
-    ipa_crud.ssh_client.run_command(command)
+    cmd_set = 'appliance_console_cli --extauth-opts="/authentication/{}=true"'.format(auth_type)
+    configured_appliance.ssh_client.run_command(cmd_set)
```

Comment:
**QUESTION:** Shouldn't we assert on the return of the command being successful?